src/status_im/ui/screens/browser/events.cljs

```diff
@@ -58,6 +61,20 @@
      (-> (add-browser-fx cofx new-browser)
          (update-in [:db :browser/options] #(assoc % :browser-id (:browser-id new-browser)))))))
 
+(handlers/register-handler-fx
+ :update-browser-with-history
+ [re-frame/trim-v]
+ (fn [{:keys [now] :as cofx} [browser url]]
+   (let [history-index (if (nil? (get browser :history-index)) 0 (get browser :history-index))
+         history (get browser :history)
+         history-url (nth history history-index)
+         browser-history-to-index (apply list (drop history-index history))
```

Comment:
Is this necessary ? `(apply list (drop history-index history))` seems the same as `(drop history-index history)`,  other that one is a `LazySeq`, or am I missing something?

Reply:
@cammellos I previously was getting a casting error, not sure what the difference was. Not needed now :+1: